src/client/sandbox/node/element.js

```diff
@@ -161,6 +162,40 @@ export default class ElementSandbox extends SandboxBase {
         var overridedGetAttributeCore    = (el, attr, ns) => this._overridedGetAttributeCore(el, attr, ns);
         var overridedSetAttributeCore    = (el, attr, value, ns) => this._overridedSetAttributeCore(el, attr, value, ns);
         var overridedRemoveAttributeCore = (el, ns, arg) => this._overridedRemoveAttributeCore(el, ns, arg);
+        var overrideInsertBefore         = (newNode, refNode, parentNode) => {
```

Comment:
Well, I believe it's a bad idea to implement `appendChild` other the `insertBefore`. We need to keep behavior as close as possible to the original, including IDL asserts